[2021-11-01 02:03:51,798] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: my_dag.create_table_passengers manual__2021-11-01T02:03:50.959281+00:00 [queued]>
[2021-11-01 02:03:51,814] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: my_dag.create_table_passengers manual__2021-11-01T02:03:50.959281+00:00 [queued]>
[2021-11-01 02:03:51,814] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-11-01 02:03:51,814] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-11-01 02:03:51,814] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-11-01 02:03:51,824] {taskinstance.py:1262} INFO - Executing <Task(PostgresOperator): create_table_passengers> on 2021-11-01 02:03:50.959281+00:00
[2021-11-01 02:03:51,828] {standard_task_runner.py:52} INFO - Started process 1564 to run task
[2021-11-01 02:03:51,831] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'create_table_passengers', 'manual__2021-11-01T02:03:50.959281+00:00', '--job-id', '75', '--raw', '--subdir', 'DAGS_FOLDER/checkModel.py', '--cfg-path', '/tmp/tmpq2o1ixn4', '--error-file', '/tmp/tmp5to9szxw']
[2021-11-01 02:03:51,831] {standard_task_runner.py:77} INFO - Job 75: Subtask create_table_passengers
[2021-11-01 02:03:51,865] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.create_table_passengers manual__2021-11-01T02:03:50.959281+00:00 [running]> on host 93c37aa78767
[2021-11-01 02:03:51,911] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=create_table_passengers
AIRFLOW_CTX_EXECUTION_DATE=2021-11-01T02:03:50.959281+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-11-01T02:03:50.959281+00:00
[2021-11-01 02:03:51,920] {base.py:79} INFO - Using connection to: id: postgres_default. Host: 172.23.0.3, Port: 5432, Schema: ***, Login: ***, Password: ***, extra: {}
[2021-11-01 02:03:51,923] {dbapi.py:225} INFO - Running statement: CREATE TABLE if not exists passengers (passengerid int, name text, sex text, age float, sibsp int, parch int, ticket text, fare float, cabin text, embarked text);, parameters: None
[2021-11-01 02:03:51,924] {postgres.py:71} INFO - NOTICE:  relation "passengers" already exists, skipping

[2021-11-01 02:03:51,934] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=create_table_passengers, execution_date=20211101T020350, start_date=20211101T020351, end_date=20211101T020351
[2021-11-01 02:03:51,962] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-11-01 02:03:51,992] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
